### DISCUSSION TITLE
Welcome 👋

### DISCUSSION THREAD

**Discussion by vault-developer** [vault-developer/event-loop-explorer#10] [General]
Hey hey, thank you for checking event-loop-explorer 👍 
I hope you find it fun and useful.
If you have any questions, suggestions or ideas feel free to raise them here and we can discuss it together. 
Happy coding! 👩‍💻 👨‍💻

**Comment by aaty2020** [comment]
hi  thanks:
I really like the simulator you're developing for understanding execution in JavaScript. I think it would be even more helpful if you added a timeline or a scrub bar to move backward and forward through the execution steps. That way, learners could better follow the flow and review specific parts more easily. Just a thought that might improve the learning experience!
